$NetBSD: patch-aa,v 1.1.1.1 1999/06/21 11:18:04 agc Exp $

Don't override PREFIX in the main Makefile, and place the
configuration files, and the executable binary, in the correct
directory, according to hier(7).

--- Makefile	1999/06/21 10:36:33	1.1
+++ Makefile	1999/06/21 10:37:26
@@ -23,7 +23,7 @@
 # The ${PREFIX} variable determines the top level directory that
 # things are installed under.
 #
-PREFIX		= /var/local
+#PREFIX		= /var/local
 
 #
 # The bin directory that user's will run the program from.
@@ -36,13 +36,13 @@
 # Directory containing sysinfo.cf configuration files
 #
 #CONFDIR		= /usr/local/lib/${PROG}
-CONFDIR		= ${PREFIX}/${PROG}/conf
+CONFDIR		= ${PREFIX}/libdata/${PROG}/conf
 
 #
 # The LIBDIR directory is where all the sysinfo binary executables
 # will be installed.
 #
-LIBDIR		= ${PREFIX}/${PROG}/lib
+LIBDIR		= ${PREFIX}/${PROG}/libexec
 
 #
 # Man sections and directories
